app-backend/database/src/main/scala/com/azavea/rf/database/tables/Buckets.scala

```diff
@@ -223,6 +242,30 @@ object Buckets extends TableQuery(tag => new Buckets(tag)) with LazyLogging {
     }
   }
 
+  /** Adds a list of scenes to a bucket
+    *
+    * @param sceneIds Seq[UUID] list of primary keys of scens to add to bucket
+    * @param bucketId UUID primary key of back to add scenes to
+    */
+  def addScenesToBucket(sceneIds: Seq[UUID], bucketId: UUID)
+    (implicit database: DB): Future[Iterable[Scene.WithRelated]] = {
+
+    database.db.run {
+      ScenesToBuckets
+        .filter(sb => sb.bucketId === bucketId && sb.sceneId.inSet(sceneIds))
+        .map(_.sceneId)
+        .result
```

Comment:
```Scala
for {
  scenebucket <- ScenesToBuckets.filter(sb => sb.bucketId === bucketId && sb.sceneId.inSet(sceneIds))
} yield (scenebucket.sceneId)
```
Think the filter + map is fine. I tend to use for comprehensions when I'm dealing with multiple tables or updates on a limited set of columns.

Anyone else have input on this?